apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>1.0.0-alpha00</Version>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <Description>ConsoleFormatter for Google Cloud Logging.</Description>
+    <PackageTags>Logging;Stackdriver;ILogger;Console;Google;Cloud</PackageTags>
```

Comment:
Remove Stackdriver? Granted people might still be using that for looking libs up.

Reply:
Yup, that was the only reason for including it. We can remove all the Stackdriver tags from all packages in a couple of years when they're no longer useful at all.